iidi/k8s/governance-ui/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - deployment.yaml
+  - service.yaml
+  - namespace.yaml
+
+configMapGenerator:
+  - name: governance-ui-config
+    literals:
+      - NEXTAUTH_SECRET=1234
+      - PT=ab
+      - SCHEMA_REGISTRY_URL=http://cp-schema-registry:8081
+      - SCHEMA_REGISTRY_FURL=http://cp-schema-registry.phac:8081
```

Comment:
I might be missing something here, but I don't think I see where these configmap values are being consumed. I believe at least `NEXTAUTH_SECRET` and `PT` are redundant here?